docs/fine-tune/index.md

```diff
@@ -0,0 +1,169 @@
+
+After your fine-tuning pod is deployed and active:
+
+1. Go to your fine-tuning pod dashboard.
+2. Click **Connect** to view connection options:
+   - **Jupyter Notebook**: Launch a browser-based notebook interface
+   - **Web Terminal**: Open a terminal in your browser
+   - **SSH**: Connect from your local machine terminal
+
+     Note: To use SSH, ensure you've added your public SSH key in your account settings. The system automatically adds your key to the pod's `authorized_keys` file.
+
+## Configure your environment
+
+When you connect to your environment, you'll find this directory structure in `/workspace/fine-tuning/`:
+
+- `examples/`: Contains sample configurations and scripts
+- `outputs/`: Stores training results and model outputs
+- `config.yaml`: Defines training parameters for your model
+
+The system generates an initial `config.yaml` based on your selected base model and dataset.
+
+## Review and modify the configuration
+
+1. Open the `config.yaml` file using a text editor:
+
+   ```bash
+   nano config.yaml
+   ```
+
+2. Review the configuration parameters and modify them as needed.
+
+   Note: It's recommended to adjust parameters based on your specific use case.
+
+Example configuration:
+
+```yaml
+base_model: NousResearch/Meta-Llama-3.1-8B
+
+load_in_8bit: false
+load_in_4bit: false
+strict: false
+
+datasets:
+  - path: tatsu-lab/alpaca
+    type: alpaca
+dataset_prepared_path: last_run_prepared
+val_set_size: 0.05
+output_dir: ./outputs/out
+
+sequence_len: 8192
+sample_packing: true
+pad_to_sequence_len: true
+
+wandb_project:
+wandb_entity:
+wandb_watch:
+wandb_name:
+wandb_log_model:
+
+gradient_accumulation_steps: 8
+micro_batch_size: 1
+num_epochs: 1
+optimizer: paged_adamw_8bit
+lr_scheduler: cosine
+learning_rate: 2e-5
+
+train_on_inputs: false
+group_by_length: false
+bf16: auto
+fp16:
+tf32: false
+
+gradient_checkpointing: true
+gradient_checkpointing_kwargs:
+  use_reentrant: false
+early_stopping_patience:
+resume_from_checkpoint:
+logging_steps: 1
+xformers_attention:
+flash_attention: true
+
+warmup_steps: 100
+evals_per_epoch: 2
+eval_table_size:
+saves_per_epoch: 1
+debug:
+deepspeed:
+weight_decay: 0.0
+fsdp:
+fsdp_config:
+special_tokens:
+  pad_token: <|end_of_text|>
+```
+
+:::note
+
+The `config.yaml` file contains all the hyper parameters needed to fine tune a model.
+You will make your adjustments in the `config.yaml`.
+
+Changes to this and running the `axolotl train config.yaml` command might require iteration.
+
+:::
+
+For more configuration examples, see the [Axolotl examples repository](https://github.com/axolotl-ai-cloud/axolotl/tree/main/examples).
+
+## Start the fine-tuning process
+
+When your configuration is ready:
+
+1. Run the following command to start fine-tuning:
+
+   ```bash
+   axolotl train config.yaml
+   ```
+
+2. The training process will begin, and progress will be displayed in your terminal.
+
+When you've finished, push your changes to Hugging Face.
```

Comment:
not worth blocking over, but is there something we can link to to describe/explain this? the biggest issue i see with fine tuning is people not knowing how to get their hands on their model.